features/com.contrastsecurity.ide.eclipse.feature/pom.xml

```diff
@@ -16,7 +16,7 @@
 	<parent>
 		<groupId>com.contrastsecurity.ide.eclipse</groupId>
 		<artifactId>features</artifactId>
-		<version>3.0.5-SNAPSHOT</version>
+		<version>3.0.6-SNAPSHOT</version>
```

Comment:
You don't have to update the versions. We can bump the version when we release a new version.

Reply:
Ok, because I didn't see a pipeline I thought that was manually updated

Reply:
Changes reverted